cloud/aws/templates/aws_oidc/bin/aws_cli.py

```diff
@@ -44,6 +44,41 @@ def update_master_password_in_database(self, db_name: str, password: str):
             f"rds modify-db-instance --db-instance-identifier={db_name} --master-user-password={password} "
         )
 
+    def sync_database_password_with_secret(self, config: ConfigLoader):
+        """
+        Updates the database password to match the value stored in the
+        secrets manager.
+        """
+        db_password = self.get_secret_value(
+            f'{config.app_prefix}-{resources.POSTGRES_PASSWORD}')
+        self.update_master_password_in_database(
+            f'{config.app_prefix}-{resources.DATABASE}', db_password)
+        print('Database password has been set.')
+
+    def set_database_credentials(
+            self,
+            config: ConfigLoader,
+            user: Optional[str] = None,
+            pwd: Optional[str] = None):
```

Comment:
(blocking) When I see pwd my first thought is "present working directory". Recommend making it password for clarity or as least pass.